cmd/soroban-cli/Cargo.toml

```diff
@@ -26,6 +26,10 @@ name = "soroban_cli"
 path = "src/lib.rs"
 doctest = false
 
+[features]
+default = []
+opt = ["wasm-opt"]
```

Comment:
Adding features means we need to start testing feature combinations in this repository. If we make `opt` the default, I think we could get away without testing these two features though. If we don't make it the default, or if we advertise building without the feature, we should test it.

Reply:
I definitely think it should be default off because we are trying to minimize the average install. But I do agree that we should have it in the released binaries.